api/v1alpha1/selfnoderemediationconfig_types.go

```diff
@@ -128,6 +128,15 @@ type SelfNodeRemediationConfigSpec struct {
 type SelfNodeRemediationConfigStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+
+	// MinSafeTimeToAssumeNodeRebootedSeconds is the minimum value that can be assigned to SelfNodeRemediationConfigSpec.SafeTimeToAssumeNodeRebootedSeconds, it is calculated and assigned dynamically.
```

Comment:
We already explain it in length in our [official docs](https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift/24.2/html/remediation_fencing_and_maintenance/self-node-remediation-operator-remediate-nodes#CO5-1), IMO because of that, it's fine to keep it short here.
However it looks like our docs could use a bit of refreshing in that context, WDYT ?

Reply:
fair enough